rest/src/test/resources/workload_rest

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2016 Yahoo! Inc. All rights reserved.                                                                                                                             

```

Comment:
Its not necessary but good to have. Since we've already mentioned a detailed README.md, this can be removed without implications.